src/lib/build/build.ts

```diff
@@ -95,7 +95,7 @@ export async function buildNexusApp(settings: BuildSettings) {
   // run of TypeScript should make re-building up this one cheap.
   tsBuilder = createTSProgram(layout, { withCache: true })
 
-  compile(tsBuilder, layout, { removePreviousBuild: true })
+  compile(tsBuilder, layout, { removePreviousBuild: false })
```

Comment:
I don't know why we did this before and I don't know why it didn't break things until now...

Reply:
I think we did this to always build from a clean slate. Why would it break? It's just like `rm -rf dist && yarn build` which is a common pattern that I believe we automated. Sounds like I'm for keeping it, but I prefer removing that kind of magic! 👍